### DISCUSSION TITLE
streampark support flink cdc 3.0?

### DISCUSSION THREAD

**Discussion by xiaolianghu** [apache/streampark#3577] [General]
flink cdc 3.0 uses yaml configuration files, is it not supported?
Reference url: https://ververica.github.io/flink-cdc-connectors/release-3.0/content/%E5%BF%AB%E9%80%9F%E4%B8%8A%E6%89%8B/mysql-doris-pipeline-tutorial-zh.html

**Comment by xiaolianghu** [comment]
![image](https://github.com/apache/incubator-streampark/assets/20347953/cc301d1e-282e-4c86-bcdc-737aaa53949a)

**Comment by caicancai** [comment]
@xiaolianghu It is currently not supported. The community is discussing 2024 RoadMap, which also supports flink-cdc3.0.
